### DISCUSSION TITLE
歌单添加不了歌曲

### DISCUSSION THREAD

**Discussion by jiuyixia** [Yin-Hongwei/vue-spring-music#175] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/105391130/167937291-f660394e-7046-4cfd-a934-7971a16cfca8.png)
大佬，这是什么问题啊。也没有提示错误

**Comment by Yin-Hongwei** [comment]
F12打开浏览器控制台看看，是最新的代码吗？

**Reply by jiuyixia** [reply]
嗯，用的最新代码
![image](https://user-images.githubusercontent.com/105391130/168066860-525497f6-f629-48b7-9d2e-d2c4508654e7.png)

**Comment by Yin-Hongwei** [comment, marked as answer]
已修复，更新一下代码。

**Reply by soo180608** [reply]
是重新下载你上传的代码然后再运行嘛

**Reply by jiuyixia** [reply]
用最新代码功能就正常了
